src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/OutlineTopComponent.java

```diff
@@ -205,6 +217,19 @@ protected boolean requestFocusInWindow(boolean temporary) {
 
     @Override
     public void resultChanged(LookupEvent lookupEvent) {
+        // Highlight the focused graph, if available, in the outline.
+        if (result.allItems().isEmpty()) {
+            return;
+        }
+        final InputGraphProvider p = LookupHistory.getLast(InputGraphProvider.class);
+        if (p == null) {
+            return;
+        }
+        try {
+            manager.setSelectedNodes(new GraphNode[]{FolderNode.getGraphNode(p.getGraph())});
```

Comment:
Do we need to consider that FolderNode.getGraphNode() returns null? `setSelectedNodes` will throw `IllegalArgumentException` if input is null?